### DISCUSSION TITLE
git repository problem?

### DISCUSSION THREAD

**Discussion by xxl2005** [vladmandic/sdnext#1621] [Q&A]
Tobias@Tobias:~
$ '/home/Tobias/Downloads/automatic-master/webui.sh' 
Create and activate python venv
Launching launch.py...
08:56:19-105903 INFO     Starting SD.Next                                                           
08:56:19-112073 INFO     Python 3.9.2 on Linux                                                      
08:56:19-113515 ERROR    Not a git repository                                                       
Tobias@Tobias:~

**Comment by Tillerz** [comment]
1. Did you copy the whole folder and forgot the (sometimes hidden) .git folder?
or
2. do instead:
```
cd /home/Tobias/Downloads/automatic-master/
./webui.sh
```
